.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 dist
+package-lock.json
```

Comment:
this project uses `yarn`, not `npm`, so why would this file ever be created? @krzkaczor what do you think?